.github/workflows/scheduled.yml

```diff
@@ -42,8 +42,37 @@ jobs:
       - name: cargo update
         if: hashFiles('server/Cargo.lock') != ''
         run: cargo update
+      - name: Download DynamoDB Local
```

Comment:
Could not run this in a docker container on CI, had network issues, so instead used an approach similar to how we do this in [faktory-rs](https://github.com/jonhoo/faktory-rs/blob/6ff7d60f2cb370c0db59c423786571d0c21bb54f/.github/workflows/ent.yaml#L23)

Reply:
That's fine :+1: